receiver/apachedruidreceiver/README.md

```diff
@@ -0,0 +1,41 @@
+# Apache Druid Receiver
+
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [development]: metrics, logs   |
+| Distributions | [] |
+| Issues        | [![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3Areceiver%2Fapachedruid%20&label=open&color=orange&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Areceiver%2Fapachedruid) [![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3Areceiver%2Fapachedruid%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aclosed+is%3Aissue+label%3Areceiver%2Fapachedruid) |
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@atoulme](https://www.github.com/atoulme), [@yuanlihan](https://www.github.com/yuanlihan) |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+<!-- end autogenerated section -->
+
+This receiver accepts [metrics](https://druid.apache.org/docs/latest/operations/metrics) data from the [HTTP Emitter](https://druid.apache.org/docs/latest/configuration/#http-emitter-module) of Apache Druid.
+
+## Prerequisites
+
+Apache Druid cluster needs to be configured to enable metrics, see [Enabling Metrics](https://druid.apache.org/docs/latest/configuration/#enabling-metrics) and the [HTTP Emitter Module](https://druid.apache.org/docs/latest/configuration/#http-emitter-module).
+
+## Configuration
+
+The following configuration options are supported:
+
+- `endpoint` (default = 0.0.0.0:9000) HTTP service endpoint for the line protocol receiver
```

Comment:
```suggestion
- `endpoint` (default = localhost:9000) HTTP service endpoint for the line protocol receiver
```
There is a discussion to move to localhost by default, 0.0.0.0 gets warnings on startup.